includes/payments/class-edd-payment.php

```diff
@@ -884,6 +886,20 @@ public function save() {
 							/** @var EDD_Discount $discount_obj */
 							$discount_obj = edd_get_discount_by( 'code', $discount );
 
+							if ( false === $discount_obj ) {
+								edd_add_order_adjustment(
+									array(
+										'object_id'   => $this->ID,
+										'object_type' => 'order',
+										'type'        => 'fee',
+										'description' => $discount,
+										'subtotal'    => floatval( $this->total - $cart_subtotal - $this->tax ),
+										'total'       => floatval( $this->total - $cart_subtotal - $this->tax ),
+									)
+								);
+								continue;
```

Comment:
Instead of `continue`, can you convert the below `edd_add_order_adjustment()` call into an `else`? I think having an if/else here provides more clarity than a continue.

Or another option would be to build up the shared arguments for `edd_add_order_adjustment()` (object_id, object_type, description). Then use an if/else to build up the conditional arguments. Then have just one `edd_add_order_adjustment()` call at the end.